doc/man7/provider-base.pod

```diff
@@ -327,7 +341,7 @@ operation C<BAR>.
 
  /* Provider context */
  struct prov_ctx_st {
-     OSSL_PROVIDER *prov;
+     OSSL_CORE_HANDLE *handle;
```

Comment:
It's interesting that we document that providers should do this, but we don't do this ourselves!